pyrostest/launch_tools.py

```diff
@@ -148,7 +148,9 @@ def new_test(self):
                     env_args=env.iteritems())
             is_master = False
             if self.port not in _LAUNCHER:
-                launch = roslaunch.scriptapi.ROSLaunch()
+                # set env variables and add argvs to sys.argv
+                os.environ['ROS_MASTER_URI'] = self.rosmaster_uri
```

Comment:
I don't think so, because we're running these in different processes, not different threads.